config/environments/test.rb

```diff
@@ -43,4 +43,7 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  # config.middleware.use RackSessionAccess::Middleware
+
+  config.cache_store = :memory_store
```

Comment:
How come we're adding this? Is this needed to get OmniAuth happy? Anything helpful we could comment this with?